assets/javascripts/modules/checkout/form-elements.js

```diff
@@ -1,36 +1,52 @@
 define(['$'], function ($) {
 
-    var formElements = {
-        $FIRST_NAME: $('.js-checkout-first'),
-        $LAST_NAME: $('.js-checkout-last'),
-        $EMAIL: $('.js-checkout-email'),
-        $ADDRESS1: $('.js-checkout-house'),
-        $ADDRESS2: $('.js-checkout-street'),
-        $ADDRESS3: $('.js-checkout-town'),
-        $POSTCODE: $('.js-checkout-postcode'),
-        $ACCOUNT: $('.js-checkout-account'),
+    return {
```

Comment:
This object looks large? Is there opportunity to narrow this to a core set of selectors, i.e., can the error handling be generalised so you don't have to handle each field on a case-by-case basis.